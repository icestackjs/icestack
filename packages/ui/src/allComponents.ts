export default [
  'alert',
  'artboard',
  'avatar',
  'badge',
  'bottomNavigation',
  'breadcrumbs',
  'button',
  'card',
  'carousel',
  'chat',
  'checkbox',
  'collapse',
  'countdown',
  'divider',
  'drawer',
  'dropdown',
  'fileInput',
  'footer',
  'form',
  'hero',
  'indicator',
  'input',
  'join',
  'kbd',
  'link',
  'mask',
  'menu',
  'mockup',
  'modal',
  'navbar',
  'progress',
  'radialProgress',
  'radio',
  'range',
  'rating',
  'select',
  'stack',
  'stat',
  'steps',
  'swap',
  'tab',
  'table',
  'textarea',
  'toast',
  'toggle',
  'tooltip'
] as const
