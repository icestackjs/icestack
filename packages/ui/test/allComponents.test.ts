import allComponents from '@/allComponents'

describe('allComponents exports', () => {
  it('snap', () => {
    expect(allComponents).toMatchSnapshot()
  })
})
