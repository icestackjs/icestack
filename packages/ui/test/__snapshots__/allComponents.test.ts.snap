// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`allComponents exports > snap 1`] = `
[
  "alert",
  "avatar",
  "button",
  "badge",
  "chat",
  "checkbox",
  "input",
  "link",
  "progress",
  "textarea",
  "toggle",
  "select",
  "radio",
  "range",
  "loading",
  "mask",
  "form",
  "table",
  "tabs",
  "skeleton",
  "radialProgress",
  "countdown",
]
`;
